server/app/core/db.py

```
@@ -0,0 +1,48 @@
+        SessionLocal = sessionmaker(
+            bind=engine,
+            autoflush=False,
+            autocommit=False,
+            expire_on_commit=False,
+            future=True,
+        )
+
+
+@contextmanager
+def get_session() -> Iterator[Session]:
+    assert SessionLocal is not None, "DB not initialized"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)